tests/main/auto-refresh-pre-download/task.yaml

```
@@ -80,8 +80,8 @@ execute: |
 
   if os.query is-ubuntu && os.query is-classic; then
     # check that the pre-download notified the user to close the snap
-    retry -n 20 MATCH 'string "Pending update of "test-snapd-sh" snap"' < /home/test/notif.log
-    MATCH 'string "Close the app to update now \(.* days left\)"' < /home/test/notif.log
+    retry -n 20 MATCH 'string "Update available for test-snapd-sh"' < /home/test/notif.log
```

This check is failing due to a missing period
```suggestion
    retry -n 20 MATCH 'string "Update available for test-snapd-sh."' < /home/test/notif.log
```